.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,25 @@
+name: Pull request
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  tests:
+    name: Test
+    uses: swiftlang/github-workflows/.github/workflows/swift_package_test.yml@macos-support
+    with:
+      linux_swift_versions: '["nightly-main"]'
```

Comment:
Do we have to specify here that we are using `swift test` and `swift build`?

Reply:
we are using `swift test`, however we can overwrite it to use `swift build`